moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -262,6 +262,9 @@ void ServoCalcs::run(const ros::TimerEvent& timer_event)
   // Do servoing calculations only if the robot should move, for efficiency
   // Create new outgoing joint trajectory command message
   auto joint_trajectory = moveit::util::make_shared_from_pool<trajectory_msgs::JointTrajectory>();
+  joint_trajectory->header.frame_id = parameters_.planning_frame;
```

Comment:
I see that these were called through `cartesianServoCals` or `jointServoCalcs` before. Why did they move up here? Is this related to the PR title?

Reply:
filling in `ros::Time::now()` here slightly changes the stamp. I believe the concrete time stamp is relevant in ros_control, so maybe we should really fill this in only right before publishing the result?

Reply:
I agree with v4hn here... the Stamp is filled at L345, right before publishing. No need to do it here.

Reply:
Removed setting the timestamp.  I moved the header frame_id and joint_names here to initialize them right after creation, I did this mostly just to make it easier to reason about it being fully initialized.